ksqldb-api-client/src/test/java/io/confluent/ksql/api/client/ClientTest.java

```diff
@@ -782,6 +782,31 @@ public void shouldHandleErrorResponseFromExecuteStatement() {
     assertThat(e.getCause().getMessage(), containsString("something bad"));
   }
 
+  @Test
+  public void shouldHandleSingleWithStatementMultipleSemicolonsFromExecuteStatement() throws Exception {
+    // Given
+    final CommandStatusEntity entity = new CommandStatusEntity(
+        "C';'SAS;",
```

Comment:
nit: make this more realistic so as to not confuse readers (else someone might see this test and wonder why we want to support this). As one example, we could have a CSAS where one of the columns being selected is a constant string containing a semicolon.